build.gradle

```diff
@@ -667,6 +675,17 @@ task genFakeSource(dependsOn: ["buildYarnTiny", "mapNamedJar"]) {
 	}
 }
 
+task decompileCFR(type: JavaExec, dependsOn: "mapNamedJar") {
+	classpath = configurations.decompileClasspath
+	main = "org.benf.cfr.reader.Main"
+
+	args namedJar.getAbsolutePath(), "--outputdir", file("src").absolutePath
```

Comment:
An option should be exposed to change the output path from something other than `./src`.

Reply:
I dont think thats needed, adds added compliexity for something that I dont think has a lot of gain.

Reply:
If there's already an existing `src`, this is what users will have to use without an option:

```bash
mv src tempSrc
./gradlew decompileCFR
mv src minecraftSrc
mv tempSrc src
```